### ISSUE TITLE
No console with recent KVM builds

### ISSUE THREAD

**Issue by andrewbogott** [andsens/bootstrap-vz#318]
My new kvm images don't produce a visible console at all.

I've been chasing this for a while.  I suspect that it's related to the console= change in 608de63d3ed4a365d6409007805d73d1f2ea909a but I don't have a smoking gun yet.

**Comment by andrewbogott** [comment]
Most likely this is related to https://github.com/andsens/bootstrap-vz/issues/312 but the associated fix doesn't seem to work for me.

**Comment by andrewbogott** [comment]
Here's a more complete statement of the problem:

I'm building on Debian GNU/Linux 8.4 using a minimal (albeit custom) manifest.  When the build finishes, I test the image like this:

 sudo qemu-system-x86_64 --nographic ./debian-jessie.raw

With the current bootstrap-vz tip code, that command sits with no feedback.  I've waited as much as 10-15 minutes to no effect.  With the attached patch, the console log starts streaming by after a few seconds, eventually ending with a login prompt.

I've also tested these images by loading into an OpenStack install using kvm.  The results are similar -- Nova is unable to provide me with a console log for default builds, but with the attached patch logging appears as I would expect.
